content/rc/security/access-control/data-access-control/role-based-access-control.md

```diff
@@ -1,6 +1,6 @@
 ---
-Title: Role-based access control
-LinkTitle: Role-based access control
+Title: Enable Role-based access control
+LinkTitle: Enable Role-based access control
```

Comment:
```suggestion
LinkTitle: Role-based access control
```
I'm okay with the title change, but the LinkTitle needs to be shorter. This one wraps around in the sidebar.

Reply:
What is the expected length of link titles?  The one before is 'Role-based access control' ..which seems pretty long.  I changed it to 'Enable Role-based access control' because I thought the pattern was to just use the same string as the Title.

Reply:
They're not always the same - for example, the [Quick start](https://docs.redis.com/latest/rc/rc-quickstart/) - you can see the title is "Redis Cloud quick start" and the LinkTitle is just "Quick start". Making them the same is the default, but if the LinkTitle wraps around in the sidebar, it's too long. That depends on how deeply the page is located.